Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -3891,6 +3891,15 @@ void MediaPlayerPrivateGStreamer::configureElement(GstElement* element)
     if (g_str_has_prefix(GST_ELEMENT_NAME(element), "brcmaudiosink")) {
         g_object_set(G_OBJECT(element), "async", TRUE, nullptr);
     }
+    if(!g_strcmp0(G_OBJECT_TYPE_NAME(G_OBJECT(element)), "GstBrcmPCMSink")) {
+#if ENABLE(MEDIA_STREAM)
+        if (player->m_streamPrivate != nullptr && g_object_class_find_property(G_OBJECT_GET_CLASS(element), "low_latency")) {
+            GST_DEBUG("Set 'low_latency' in brcmpcmsink");
+            g_object_set (G_OBJECT(element), "low_latency", TRUE, nullptr);
```

Comment:
If you feel like and also think that this makes things simple, you can set the two properties at the same time with a single `g_object_set(object, "property1", value1, "property2", value2, ..., nullptr)` call. Do it or not at your own discretion.